src/component/project/Project/ProjectOverview.tsx

```diff
@@ -28,7 +28,7 @@ const ProjectOverview = ({ projectId }: IProjectOverviewProps) => {
                     health={health}
                     featureCount={features?.length}
                 />
-                <div className={styles.projectToggles}>
+                <main className={styles.projectToggles}>
```

Comment:
There's probably already a `<main>` further up the tree, from `MainLayout`. Should only have one.